README.md

```diff
@@ -276,37 +276,102 @@ To build LPVS from source code and run it in the default service mode, follow th
 
 LPVS is now built and running. You can create a new pull request or update an existing one with commits, and LPVS will automatically start scanning and provide comments about the licenses found in the project.
 
-#### 4.b Single scan mode
+#### 4.2 Single scan mode
 
 Alternatively, you can perform a one-time scan on a specific pull request using the single scan mode. Follow these steps:
 
-4.b.1. Begin by running the installation and navigating to the target directory, similar to the process in service mode (refer to steps 4.a.1 and 4.a.2):
+4.2.1 Begin by running the installation and navigating to the target directory, similar to the process in service mode (refer to steps 4.2.1 and 4.2.2):
 
    ```bash
    mvn clean install
    cd target/
    ```
 
-4.b.2. Execute the single scan with the following command:
+4.2.2 Execute the single scan with the following command:
 
    ```bash
    java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL>
    ```
 
-4.b.3. By default, the above command requires a pre-configured MySQL database. To avoid setting up the database, use the "singlescan" profile:
+4.2.3 By default, the above command requires a pre-configured MySQL database. To avoid setting up the database, use the "singlescan" profile:
    ```bash
    java -jar -Dspring.profiles.active=singlescan -Dgithub.token=<my-token> lpvs-*.jar --github.pull-request=<PR URL>
    ```
 
 These steps streamline the process, allowing you to run a scan on a single pull request without the need for a preconfigured database.
 
-4.b.4. Available option to generate an HTML report and save it in a specified folder. Replace `/path/to/your/folder` with the full path to the folder where you want to save the HTML report, and `your_report_filename.html` with the desired filename for the report.
+4.2.4 Available option to generate an HTML report and save it in a specified folder. Replace `/path/to/your/folder` with the full path to the folder where you want to save the HTML report, and `your_report_filename.html` with the desired filename for the report.
    ```bash
    java -jar -Dgithub.token=<my-token> lpvs-*.jar --github.pull.request=<PR URL> --build.html.report=</path/to/your/folder/your_report_filename.html>
    ```
 
 These steps streamline the process, allowing you to run a scan on a single pull request without the need for a preconfigured database.
 
+#### 4.3 Use of lpvs-x.y.z.jar in your project
```

Comment:
4.3 Use of LPVS JAR `lpvs-x.y.z.jar` in your project

Reply:
Done!